var/spack/repos/builtin/packages/sgpp/set_boost_lib_path_internally.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/SConstruct b/SConstruct
```

Comment:
Just in case this diff comes from an existing commit, I like to reference the patch from github, include a checksum and just get it from there, as it gives 1. context to the change and 2. doesn't need to be included here.

It's a bit subjective what's nicer though, so this is also just a suggestion.